tests/test_optional_extras.py

```diff
@@ -0,0 +1,70 @@
+from app.models.goal import Goal
```

Comment:
Yeah! I like all of the tests in this file. Though I did notice something naming wise about your API design...